Added new operators: `in`, `in caseless`, `not in` and `not in caseless`. (#13144)
demisto/content: Packs/ExtFilter/ReleaseNotes/1_1_4.md

#### Scripts
##### ExtFilter

> - Fixed an issue with causing `Circular reference detected` with using `is updated with`.

Reviewer: ```suggestion
- Fixed an issue where the `is updated with` operator was causing circular references. 
```

> - Fixed an issuw that `contains` doesn't work for non-string values.

Reviewer: ```suggestion
- Fixed an issue where the 'contains' opertator would not work for non-string values.
```
